src/nonFree/sift/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # libs should be static
-set(BUILD_SHARED_LIBS ON)
```

Comment:
Is there a reason to override the project value? Could we not just remove this line?

Reply:
This is because `alicevision_add_library` uses this to decide whether to build static or shared library. Other dependencies are handled in a similar way by including them as static libraries. I think we should do the same in this case too.

Reply:
@fabiencastan Just a friendly ping :-)